openstf/model/confidence_interval_applicator.py

```diff
@@ -2,18 +2,45 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 import numpy as np
+import pandas as pd
 from scipy import stats
 
 
+class ConfidenceIntervalApplicatorBaseCase:
+    def __init__(self, model):
+        self.confidence_interval = model.confidence_interval
+
+    def add_confidence_interval(self, basecase_forecast: pd.DataFrame) -> pd.DataFrame:
+        """Adds confidence interval to basebase forecast
```

Comment:
Could you add some docstring to explain how ConfidenceIntervalApplicatorBaseCase differs from ConfidenceIntervalApplicator? Would help the next reader:)